### DISCUSSION TITLE
How to run an extension from command line?

### DISCUSSION THREAD

**Discussion by wiiznokes** [microsoft/vscode-discussions#1069] [Extension Development QnA] [answered]
I would like an equivalent of f5 to run my extension. It doesn't always works and i prefer using the command line for this type of stuff. I would also like to know if it's possible to automatically open a folder when testing the extension.
And finally, it is possible to have the extension log output in the initial vscode windows instead of the one being tested ?

**Comment by isidorn** [comment, marked as answer]
Not possible to run F5 from the command line.
Auto open a folder - your extension could can do that. Just use vscode api.

For log output, try the LogOutputChannel API https://code.visualstudio.com/api/references/vscode-api#LogOutputChannel
